x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout/activity_item.tsx

```diff
@@ -71,7 +71,10 @@ export const ActivityItem: React.FunctionComponent<{
       id="xpack.fleet.agentActivityFlyout.completedDescription"
       defaultMessage="Completed {date}"
       values={{
-        date: formattedTime(action.completionTime),
+        date:
+          action.completionTime === '0001-01-01T00:00:00.000Z'
```

Comment:
interesting, it sounds like a bug, maybe we could add a comment where this value is coming from, if we want to investigate later